dts/arm/nxp/nxp_rt.dtsi

```diff
@@ -800,19 +799,27 @@
 			#pinmux-cells = <2>;
 			reg = <0x40384000 0x4000>;
 			clocks = <&ccm IMX_CCM_SAI1_CLK 0x7C 2>;
-			pre-div = <0>;
-			podf = <63>;
-			pll-clocks = <&anatop 0x70 0xC000 0>,
-					<&anatop 0x70 0x7F 32>,
-					<&anatop 0x70 0x18 1>,
-					<&anatop 0x80 0x3FFFFFFF 77>,
-					<&anatop 0x90 0x3FFFFFFF 100>;
+			// Audio PLL Output Frequency is determined by:
+			// (Fref * (DIV_SELECT + NUM/DENOM)) / POST_DIV
+			// = (24MHz * (32 + 77 / 100)) / 1 = 786.48 MHz
+		       pll-clocks = <&anatop 0x70 0xC000 0>,
```

Comment:
although defined in device tree, but driver code does not use it, still default value